internal/cmsapi/password_recovery.go

```diff
@@ -0,0 +1,95 @@
+package cmsapi
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+const (
+	PathAccountPasswordRecoveryEmailGet = "/account/password/recovery/email/get"
+	PathAccountPasswordRecoveryEmailSet = "/account/password/recovery/email/set"
+)
+
+type responseStatus struct {
+	ErrorCode string
+	Message   string
+}
+
+func (r responseStatus) errorOrNil() error {
+	if (r == responseStatus{}) {
+		return nil
+	}
+	return &r
+}
+
+func (r *responseStatus) Error() string { return r.Message }
+
+func PasswordRecoveryEmailGet(ctx context.Context, callsign, password string) (string, error) {
+	req := newJSONRequest("GET", PathAccountPasswordRecoveryEmailGet,
+		url.Values{"callsign": []string{callsign}, "password": []string{password}},
+		nil).WithContext(ctx)
+	resp, err := http.DefaultClient.Do(req)
```

Comment:
Might be worth setting a custom User Agent when sending this. I think we do so in other places...